### DISCUSSION TITLE
Welcome to MahApps.Metro Discussions!

### DISCUSSION THREAD

**Discussion by punker76** [MahApps/MahApps.Metro#3997] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * The [Gitter](https://gitter.im/MahApps/MahApps.Metro) room is still available until we see that discussions is a better place.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community and MahApps.

**Comment by timunie** [comment]
Hi together,

My name is Tim and I live in Germany. I am an engineer by profession and like to do some more or less "simple" programming in my free time. I also use my programming skills to make my work easier, as an engineer is a really lazy animal. 😆 

![image](https://user-images.githubusercontent.com/47110241/101630410-1f665a80-3a23-11eb-9961-93486e8de403.png)

I use MahApps as my basic GUI-Framework as it is beautiful and quite powerful out of the box. From time to time I also add some new features or help with bug fixes. Moreover I like to help others getting started with MahApps whenever I can. 

Happy coding,
Tim
